spec/shared/helpers/application_helper/buttons/basic.rb

```diff
@@ -0,0 +1,5 @@
+shared_context 'ApplicationHelper::Button::Basic' do
+  let(:view_context) { setup_view_context_with_sandbox(sandbox) }
+  let(:button) { described_class.new(view_context, {}, instance_data, props) }
```

Comment:
Need to use `let(:button)` due to some shared examples that depend on the existence of the `button` method.
Shall be removed in the last refactoring commit.